config/config.go

```diff
@@ -154,11 +154,11 @@ func init() {
 	cfg.SetDefault("storage.orientdb.password", "root")
 
 	cfg.SetDefault("ui.bandwidth_absolute_active", 1)
-	cfg.SetDefault("ui.bandwidth_absolute_alert", 1000)
 	cfg.SetDefault("ui.bandwidth_absolute_warning", 100)
+	cfg.SetDefault("ui.bandwidth_absolute_alert", 1000)
```

Comment:
please do not change the order, it was in alphabetical order

Reply:
ok